authors/apps/articles/models.py

```diff
@@ -59,7 +57,7 @@ class Article(models.Model):
     description = models.CharField(max_length=230, blank=False)
     body = models.TextField(blank=False)
     created_at = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateField(auto_now=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Was this the area @kwanj-k 's [Heroku runtime error](https://github.com/andela/ah-the-unsullied/pull/22#pullrequestreview-198221972) referring to? If so, why did we resolve to leaving it as it is?